guacamole/src/main/frontend/src/translations/fr.json

```diff
@@ -21,26 +27,33 @@
         "ACTION_MANAGE_USER_GROUPS" : "Groupes",
         "ACTION_NAVIGATE_BACK"      : "Retour",
         "ACTION_NAVIGATE_HOME"      : "Accueil",
+        "ACTION_PAUSE"              : "Pause",
+        "ACTION_PLAY"               : "Lecture",
         "ACTION_SAVE"               : "Enregistrer",
         "ACTION_SEARCH"             : "Rechercher",
         "ACTION_SHARE"              : "Partager",
         "ACTION_UPDATE_PASSWORD"    : "Mettre à jour mot de passe",
         "ACTION_VIEW_HISTORY"       : "Historique",
+        "ACTION_VIEW_RECORDING"     : "Voir",
 
         "DIALOG_HEADER_ERROR" : "Erreur",
 
-        "ERROR_PAGE_UNAVAILABLE"  : "Une erreur est apparue et cette action ne pourra pas être achevé. Si le problème persiste, merci de contacter votre administrateur ou regarder les journaux système.",
+        "ERROR_PAGE_UNAVAILABLE"  : "Une erreur est survenue et cette action ne pourra pas être achevée. Si le problème persiste, merci de contacter votre administrateur ou regarder les journaux système.",
         "ERROR_PASSWORD_BLANK"    : "Votre mot de passe ne peut pas être vide.",
         "ERROR_PASSWORD_MISMATCH" : "Le mot de passe ne correspond pas.",
+        "ERROR_SINGLE_FILE_ONLY"  : "Veuillez télécharger un seul fichier à la fois.",
 
         "FIELD_HEADER_PASSWORD"       : "Mot de passe:",
         "FIELD_HEADER_PASSWORD_AGAIN" : "Répéter mot de passe:",
+        "FIELD_HEADER_RECORDING_WRITE_EXISTING" : "Autoriser l'écriture dans le fichier d'enregistrement existant:",
+        "FIELD_HEADER_TYPESCRIPT_WRITE_EXISTING" : "Autoriser l'écriture dans le fichier TypeScript existant:",
```

Comment:
Would be best, here, to align the ":" in these two lines, following established style throughout these translation files. I think you've done that most everywhere else, so I think this may have just been a miss.

Reply:
Modified